### PR TITLE
Expand Conversations on Inbox page

### PR DESCRIPTION
**Pull request by Raemon**
Normally the inbox page looks like this:

image

I needed to process a bunch of PMs at once, so I made myself a version of that you can expand to look like this:

image

Previously, the Inbox loaded 200 conversations, and didn't have a load more at the bottom. Expanding all 200 convos took awhile, and I don't think anyone would ever need them. So I changed this around so it loads 50, and _does_ have a load more at the bottom.

### CODE REVIEW

File: packages/lesswrong/components/localGroups/LocalGroupsList.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/localGroups/LocalGroupsList.tsx:6]: I think we should try to avoid sharing css this way. I don't like the way that often in codebases, you want to change some CSS, and then you need to check a bunch of places to see that you haven't broken anything. The way that the react ecosystem solves that is by tying css to components.

Obviously some things like flexGrow shouldn't be tied to the component. But I would just copy and paste those.

I'm not sure about the purpose of the pseudo-element styles, but I wonder if they can be put on the component?

If you really want to share this CSS, but not put it on the component, please move this declaration to the LoadMore component.

**Comment by jpaddison3** [comment]
The LoadMore loading state pulls itself to the right where it collides with the "Show archived conversations" tickbox and looks terrible.

File: packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx:66]: Please remove this file's diff from this PR

Comment (jpaddison3) [reply packages/lesswrong/components/review/ReviewVotingExpandedPost.tsx:66]: Which might just happen on merging, I haven't checked.

**Review by jpaddison3 — APPROVED** [review]
Good work building things when you notice things you want. ASTLAC.

Please address comments before merging, but I trust I won't need to do another review.

File: packages/lesswrong/components/localGroups/LocalGroupsList.tsx
Comment (Raemon) [reply packages/lesswrong/components/localGroups/LocalGroupsList.tsx]: Hmm. Currently our paradigm seemed to be "if you're going to copypaste some CSS  from one place to another, better to export it this way so things don't get out of sync."

I do think that doing this is usually a warning flag that your component isn't architected right (and I agree it is in this case). I went and changed it. I'm not sure whether I endorse making this sort of incremental improvement is worth the slowdown of random PRs that were mostly not about this.

(The status quo is that there were several different components that had a similar style, and it seemed to me that getting them synced up was a marginal improvement. I went and consolidated all of them into the LoadMore component, which wasn't really the point of this PR, which wasn't a very important PR to begin with, but seems like this part of the codebase was due for a cleanup ayway)

**Pull request merged**
